bin/runtime/src/lib.rs

```diff
@@ -107,7 +107,7 @@ pub const VERSION: RuntimeVersion = RuntimeVersion {
     spec_name: create_runtime_str!("aleph-node"),
     impl_name: create_runtime_str!("aleph-node"),
     authoring_version: 1,
-    spec_version: 26,
+    spec_version: 27,
```

Comment:
needed? if so why?

Reply:
as in the PR description:

> I bumped spec_version as well, since all pallets depend on the changed dependency (tracing-related) so as their logging part.

this potentially can change wasm bytecode